satellite-software/src/LSM9DS1Sensor.cpp

```diff
@@ -349,4 +349,46 @@ void LSM9DS1Sensor::setGysroSensitivitySetting(GyroSensitivitySetting setting) {
 void LSM9DS1Sensor::setMagneticSensitivitySetting(MagneticSensitivitySetting setting) {
     eeprom_update_byte((uint8_t *)(getEEPROMBaseAddress()+OFFSET_MAGNETIC_SENSITIVITY), setting);
     _magneticSensitivity = setting;
+}
+
+uint8_t LSM9DS1Sensor::handleCommand(uint16_t cmdSequenceID, uint8_t command, uint8_t* recievedData, uint8_t recievedDataLen){
+    if(recievedDataLen != 1){
+        return CMD_STATUS_BAD_PARAM;
+    }
+    PRINT_DEBUG(F("Change LSM9DS1 settings"));
+    PRINT_DEBUG(F("\n"));
+    
+    switch(command){
```

Comment:
Please fix indentation to match how indentation for switch statements are used elsewhere in this project. Also, please match how spaces are used in `if` statements.

Reply:
Done